bundles/net.tourbook.device.garmin/src/net/tourbook/device/garmin/GarminSAXHandler.java

```diff
@@ -33,12 +34,14 @@
 import net.tourbook.common.util.Util;
 import net.tourbook.data.TimeData;
 import net.tourbook.data.TourData;
+import net.tourbook.data.TourTimerPause;
 import net.tourbook.importdata.DeviceData;
 import net.tourbook.importdata.TourbookDevice;
 import net.tourbook.tour.TourLogManager;
 
 import org.eclipse.jface.preference.IPreferenceStore;
 import org.eclipse.osgi.util.NLS;
+import org.joda.time.DateTime;
```

Comment:
Ok, good idea, I will remove this joda time dependency and will try to consider that in other parts of codes in the future.
I used Joda for that :
`DateTime.parse(startTimeValue);`

but i will try to find a java equivalent.

I think it will be easier to get rid of libraries like Joda when we move to Java 11 ;-)

Reply:
Fixed